src/import-data/ImportDataDestination.test.ts

```diff
@@ -114,8 +114,8 @@ describe('ImportDataDestination', () => {
       });
 
       // Act
-      const existingWorkspace = screen.getByText(selectExistingWorkspacePrompt, { exact: false });
-      await user.click(existingWorkspace); // select start with existing workspace
+      const existingWorkspaceButton = screen.getByText(selectExistingWorkspacePrompt, { exact: false });
```

Comment:
This rename is split out into its own [commit](f1c4b8cfb5eaf317768a5f658013bdbf1f49e9bf) for easier review.